src/services/jsDoc.ts

```diff
@@ -258,17 +259,21 @@ export function getJsDocTagsFromDeclarations(declarations?: Declaration[], check
         }
         for (const tag of tags) {
             infos.push({ name: tag.tagName.text, text: getCommentDisplayParts(tag, checker) });
-
-            if (isJSDocPropertyLikeTag(tag) && tag.isNameFirst && tag.typeExpression && isJSDocTypeLiteral(tag.typeExpression.type)) {
-                forEach(tag.typeExpression.type.jsDocPropertyTags, propTag => {
-                    infos.push({ name: propTag.tagName.text, text: getCommentDisplayParts(propTag, checker) });
-                });
-            }
+            infos.push(...getJSDocPropertyTagsInfo(tag, checker));
         }
     });
     return infos;
 }
 
+function getJSDocPropertyTagsInfo(node: JSDocTag, checker: TypeChecker | undefined): readonly JSDocTagInfo[] {
+    return flatMap(tryGetJSDocPropertyTags(node), propTag => concatenate([{ name: propTag.tagName.text, text: getCommentDisplayParts(propTag, checker) }], tryGetJSDocPropertyTags(propTag) ? getJSDocPropertyTagsInfo(propTag, checker) : undefined));

```

Comment:
```suggestion
function getJSDocPropertyTagsInfo(nodes: JSDocTag[] | undefined, checker: TypeChecker | undefined): readonly JSDocTagInfo[] {
    return flatMap(nodes, propTag => concatenate([{ name: propTag.tagName.text, text: getCommentDisplayParts(propTag, checker) }],  getJSDocPropertyTagsInfo(tryGetJSDocPropertyTags(propTag), checker) : undefined));

```

Reply:
Thanks. I've added the requested changed.